fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -636,6 +636,7 @@ public void addPartitionsCache(String dbName, String tblName, List<String> parti
         HivePartitionValues copy = partitionValues.copy();
         Map<Long, PartitionItem> idToPartitionItemBefore = copy.getIdToPartitionItem();
         Map<String, Long> partitionNameToIdMapBefore = copy.getPartitionNameToIdMap();
+        Map<Long, String> partitionIdToNameMapBefore = copy.getPartitionIdToNameMap();
```

Comment:
both id and name should be unique, may be better to use bimap to avoid inconsistent bewteen them